mongomock/helpers.py

```diff
@@ -418,3 +419,11 @@ def mongodb_to_bool(value):
     """Converts any value to bool the way MongoDB does it"""
 
     return value not in [False, None, 0]
+
+
+def to_long(value):
+    """Backwards compatible `long` function. It tries to convert input to a long integer."""
+    if sys.version_info >= (3, 0):
+        global long  # pylint: disable=global-variable-undefined
+        long = int
+    return long(value)
```

Comment:
I'm not sure why codecov is complaining that this is not covered by test. Can you quickly check please?